prov/efa/src/efa_strerror.c

```diff
@@ -51,11 +51,19 @@ static const char *efa_io_comp_status_str(enum efa_errno status)
 		[FI_EFA_LOCAL_ERROR_BAD_LENGTH]        = "Message too long",
 		[FI_EFA_REMOTE_ERROR_BAD_ADDRESS]      = "Invalid address",
 		[FI_EFA_REMOTE_ERROR_ABORT]            = "Receiver connection aborted",
-		[FI_EFA_REMOTE_ERROR_BAD_DEST_QPN]     = "Invalid receiver queue pair number (QPN)",
+		[FI_EFA_REMOTE_ERROR_BAD_DEST_QPN]     = "Invalid receiver queue pair number (QPN). "
+		                                         "This error can be caused by a crashed peer. "
```

Comment:
I think we need stronger word. Instead  of "can be", I think we use "is typically"